.gitignore

```diff
@@ -8,15 +8,16 @@
 /clusterctl
 /cluster-api-provider-vsphere*
 
-# Generated files
-/cluster-api-provider-vsphere
+# Examples
 /examples/provider-components/provider-components*.yaml
-test/e2e/data/infrastructure-vsphere/kustomization/base/cluster-template.yaml
-test/e2e/data/infrastructure-vsphere/kustomization/topology/cluster-template-topology.yaml
-test/e2e/data/infrastructure-vsphere/cluster-template*.yaml
-test/e2e/data/infrastructure-vsphere/clusterclass-quick-start.yaml
+
+# E2E test artifacts folder
 _artifacts/
 
+# E2E test templates
+test/e2e/data/infrastructure-vsphere/main/**/cluster-template*.yaml
```

Comment:
Can probably do something smarter here - will think about it when I add additional versions to this test.

Reply:
This is what we use in core CAPI
```
test/e2e/data/infrastructure-docker/**/cluster-template*.yaml
test/e2e/data/infrastructure-inmemory/**/cluster-template*.yaml
```